js/actions/appActions.js

```diff
@@ -1411,6 +1411,14 @@ const appActions = {
     dispatch({
       actionType: appConstants.APP_UPDATE_LOG_OPENED
     })
+  },
+
+  updateComputedStyles: function (windowId, values) {
```

Comment:
also should be declarative `updateComputedStyles` -> `computedStylesChanged` or something similar